lib/src/popup_marker_layer_widget.dart

```diff
@@ -31,33 +31,30 @@ class _PopupMarkerLayerWidgetState extends State<PopupMarkerLayerWidget> {
   Widget build(BuildContext context) {
     return PopupStateWrapper(
       builder: (context, popupState) => _layers(
-        mapState: flutter_map.MapState.maybeOf(context)!,
+        mapState: flutter_map.FlutterMapState.maybeOf(context)!,
```

Comment:
Maybe now that `MapState` is renamed to `FlutterMapState` we don't need to disambiguate with flutter_map.FlutterMapState?

Reply:
Still need to with your custom MarkerLayer class.

Reply:
Unless we remove the custom MarkerLayer Widget or change its name we need to keep the disambiguation.